repl/src/main/scala/ammonite/repl/Main.scala

```diff
@@ -77,15 +80,17 @@ case class Main(predef: String = "",
     */
   def runScript(path: Path,
                 args: Seq[String],
-                kwargs: Map[String, String]): Res[Imports] = {
+                kwargs: Map[String, String],
+                storage: Storage): Res[Seq[ImportData]] = {
```

Comment:
Also, why are we changing the signature from `Res[Imports]` to `Res[Seq[ImportData]]`? That should not be necessary